libs/efivar/patches/006-fix-lib-path.patch

```diff
@@ -0,0 +1,10 @@
+--- a/src/include/defaults.mk
++++ b/src/include/defaults.mk
+@@ -1,6 +1,6 @@
+ PREFIX	?= /usr
+ EXEC_PREFIX ?= $(PREFIX)
+-LIBDIR	?= $(PREFIX)/lib64
++LIBDIR	?= $(PREFIX)/lib
```

Comment:
replaced by `LIBDIR="/usr/lib"`. can be removed.

Reply:
patch removed